client/lib/abtest/active-tests.js

```diff
@@ -8,6 +15,21 @@ module.exports = {
 		defaultVariation: 'disabled',
 		allowExistingUsers: true,
 	},
+	// `browserNotificationsPreferences` controls whether or not users see the
+	// preference to enable browser notifications in /me/notifications;
+	// any users with `browserNotifications` enabled will also see the preference;
+	// this is a temporary test just to allow us to ramp up the load gradually so
+	// that make sure things will be smooth when we go to launching browser
+	// notifications for 100% of users
+	browserNotificationsPreferences: {
+		datestamp: '20160801',
+		variations: {
+			disabled: 0,
```

Comment:
Oh wow... don't know what I was thinking here... certainly! Fixed! (I'm pretty sure that would have enabled things for 100% of users in this test.)